modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/PublicationTimeline.js

```diff
@@ -116,43 +84,36 @@ const PublicationTimeline = ({namespace, timelineItemsURL}) => {
 								</ClayLayout.ContentCol>
 
 								<ClayLayout.ContentCol>
-									{timelineItem.actions ? (
-										<TimelineDropdownMenu
-											deleteURL={
-												timelineItem.status.code ===
-													WORKFLOW_STATUS_DRAFT &&
-												!!timelineItem.actions.delete
-													? timelineItem.actions
-															.delete.href
-													: undefined
-											}
-											editURL={
-												timelineItem.status.code ===
-													WORKFLOW_STATUS_DRAFT &&
-												!!timelineItem.actions.update
-													? getEditURL(
-															timelineItem.id
-														)
-													: undefined
-											}
-											revertURL={
-												timelineItem.status.code ===
-												WORKFLOW_STATUS_APPROVED
-													? getRevertURL(
-															timelineItem.id
-														)
-													: undefined
+									{timelineItem.actions.get ? (
+										<ClayDropDown
+											alignmentPosition={Align.BottomLeft}
+											renderMenuOnClick
+											spritemap={spritemap}
+											trigger={
+												<ClayButtonWithIcon
+													aria-label="timeline-actions"
+													displayType="unstyled"
+													small
```

Comment:
`small` is marked as deprecated. Can you switch it to `size="sm"` so we don't need to remember to change it down the line?